### PR TITLE
update coalesceFindRequests doc for JSONAPIAdapter

### PR DESCRIPTION
**Pull request by drewclauson**
Replacement for #3770

### CODE REVIEW

File: packages/ember-data/lib/adapters/json-api-adapter.js [hunk 1]
```diff
@@ -39,6 +39,55 @@ export default RESTAdapter.extend({
     return hash;
   },
 
+    /**
```
Comment (bmac) [inline packages/ember-data/lib/adapters/json-api-adapter.js]: The linter wants this line to start with 2 spaces not 3.

File: packages/ember-data/lib/adapters/json-api-adapter.js [hunk 2]
```diff
@@ -39,6 +39,55 @@ export default RESTAdapter.extend({
...
+    will also send a request to: `GET /comments?filter[id]=1,2`
+
+    Note: Requests coalescing rely on URL building strategy. So if you override `buildURL` in your app
+    `groupRecordsForFindMany` more likely should be overridden as well in order for coalescing to work.
+
+    @property coalesceFindRequests
+    @type {boolean}
+  */
+  coalesceFindRequests: false,
+
```
Comment (bmac) [inline packages/ember-data/lib/adapters/json-api-adapter.js]: The linter also wants the whitespace removed here. 

@drewclauson after you make these changes locally you can stage them (`git add packages/ember-data/lib/adapters/json-api-adapter.js`) then amend the existing commit using `git commit --amend` then force push an update to this pr using `git push -f`.

Comment (drewclauson) [inline packages/ember-data/lib/adapters/json-api-adapter.js]: Thanks for the steps, that's a lot easier than what I was trying with rebase and other things I found online!

**Comment by bmac** [comment]
Thanks @drewclauson.

**Pull request merged**